package.json

```diff
@@ -112,7 +113,7 @@
   "bundlesize": [
     {
       "path": "./dist/DO-NOT-USE.min.js",
-      "threshold": "3 Kb"
+      "threshold": "6 Kb"
```

Comment:
Keep in mind this is my first pass at getting glamor's core algo working here. I keep finding code from features that I don't need and removing it.